src/patterns/components/awards/primary.hbs

```diff
@@ -26,7 +26,7 @@ hasAngularCodeInfo: true
     </a>
   </div>
 
-  <div class="sprk-o-Stack__item sprk-o-Stack__item--start-column" data-sprk-toggle="container">
+  <div class="sprk-o-Stack__item sprk-o-Stack__item--stretch-column" data-sprk-toggle="container">
```

Comment:
Hey @janinavelasco9, thanks for the PR! I had one change request for this. Align stretch is the flexbox default and in order to avoid repeating, we can remove the Stack parent's center class (`sprk-o-Stack--center-column`) and then allow the items inside to be set individually. For each item we would add `sprk-o-Stack__item--center-column` but omit that for the final disclaimer element which we want to have the defaults. Then we wouldn't need the stretch classes.

Reply:
yes! i actually saw that stretch was the default and was making a similar change but then had to run to the gym. but what you said makes perfect since! ill make the change.

Reply:
Thanks! Appreciate the extra help so much!